src/lib/renderComponent.tsx

```diff
@@ -221,6 +224,10 @@ const renderComponent = <P extends {}>(config: RenderConfig<P>): React.ReactElem
         const classes: ComponentSlotClasses = getClasses(renderer, mergedStyles, styleParam)
         classes.root = cx(className, classes.root, props.className)
 
+        const { children, content } = props
+        const evaluatedChildren = getRtlAwareElement(children)
```

Comment:
may suggest to rename these two to something that is less abstract in terms of providing their intent (as `evaluatedChildren` is quite general). Think that something with `rtl` in its name would work better

Reply:
How about `rtlTransformedChildren` & `rtlTransformedContent`?